cmake/rocprim-backward-compat.cmake

```diff
@@ -0,0 +1,309 @@
+
+#Include Directory Paths for - wrapper.
+set(PROJECT_WRAPPER_TEMPLATE_FILE       ${PROJECT_SOURCE_DIR}/header.hpp.in)
+set(PROJECT_WRAPPER_MAIN_INC_DIR        ${PROJECT_BINARY_DIR}/include/rocprim)
+set(PROJECT_WRAPPER_INC_BLOCK           ${PROJECT_BINARY_DIR}/include/rocprim/block)
+set(PROJECT_WRAPPER_INC_DETAIL          ${PROJECT_BINARY_DIR}/include/rocprim/detail)
+set(PROJECT_WRAPPER_INC_DEVICE          ${PROJECT_BINARY_DIR}/include/rocprim/device)
+set(PROJECT_WRAPPER_INC_INTRINSICS      ${PROJECT_BINARY_DIR}/include/rocprim/intrinsics)
+set(PROJECT_WRAPPER_INC_ITERATOR        ${PROJECT_BINARY_DIR}/include/rocprim/iterator)
+set(PROJECT_WRAPPER_INC_TYPES           ${PROJECT_BINARY_DIR}/include/rocprim/types)
+set(PROJECT_WRAPPER_INC_THREAD          ${PROJECT_BINARY_DIR}/include/rocprim/thread)
+set(PROJECT_WRAPPER_INC_WARP            ${PROJECT_BINARY_DIR}/include/rocprim/warp)
+set(PROJECT_WRAPPER_INC_BLK_DET         ${PROJECT_BINARY_DIR}/include/rocprim/block/detail)
+set(PROJECT_WRAPPER_INC_DEV_DET         ${PROJECT_BINARY_DIR}/include/rocprim/device/detail)
+set(PROJECT_WRAPPER_INC_DEV_SPE         ${PROJECT_BINARY_DIR}/include/rocprim/device/specialization)
+set(PROJECT_WRAPPER_INC_ITE_DET         ${PROJECT_BINARY_DIR}/include/rocprim/iterator/detail)
+set(PROJECT_WRAPPER_INC_WAR_DET         ${PROJECT_BINARY_DIR}/include/rocprim/warp/detail)
+
+
+#Source Include File Directories, Level1 and Level2 Subdirectories
+set(PROJECT_FREORG_MAIN_INC_DIR         ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim)
+set(PROJECT_FREORG_INC_BLOCK            ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/block)
+set(PROJECT_FREORG_INC_DETAIL           ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/detail)
+set(PROJECT_FREORG_INC_DEVICE           ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/device)
+set(PROJECT_FREORG_INC_INTRINSICS       ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/intrinsics)
+set(PROJECT_FREORG_INC_ITERATOR         ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/iterator)
+set(PROJECT_FREORG_INC_TYPES            ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/types)
+set(PROJECT_FREORG_INC_THREAD           ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/thread)
+set(PROJECT_FREORG_INC_WARP             ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/warp)
+set(PROJECT_FREORG_INC_BLK_DET          ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/block/detail)
+set(PROJECT_FREORG_INC_DEV_DET          ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/device/detail)
+set(PROJECT_FREORG_INC_DEV_SPE          ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/device/specialization)
+set(PROJECT_FREORG_INC_ITE_DET          ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/iterator/detail)
+set(PROJECT_FREORG_INC_WAR_DET          ${PROJECT_SOURCE_DIR}/rocprim/include/rocprim/warp/detail)
+
+#Relative Path for wrapper (include) files generated 
+set(PROJECT_WRAPPER_MAIN_INC_RELPATH    "${include_statements}#include \"../../../include/rocprim")
```

Comment:
The `include_statements` here aren't set. What is the point of having this variable in these strings?

Reply:
Updated in the latest version